src/plugins/action_files_cleanup/src/cleanupFiles.ts

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+import databaseMethods from './databaseMethods'
+
+const fileUploadPluginCode = 'fileUpload'
+
+interface FileInfo {
+  uniqueId: string
+  applicationResponseId: number
+  filePath: string
+  thumbnailPath: string
+}
+
+interface ResponseValue {
+  text?: string
+  files?: {
+    fileUrl: string
```

Comment:
only one of these is used, do we envision others being used ?

Reply:
No, where not using them here, I was just being strict about defining what is actually going to be returned from the database (that's the shape of the JSON object in "value" for file responses).